website/screens/components/checkbox/code/examples/uncontrolled.ts

```diff
@@ -1,23 +1,44 @@
 import {
   DxcCheckbox,
   DxcInset,
+  DxcButton,
+  DxcFlex,
 } from "@dxc-technology/halstack-react";
+import { useRef } from "react";
 
 const code = `() => {
-  const onChange = (newValue) => {
-    console.log(newValue);
+  const checkboxRef = useRef();
+
+  const handleSubmit = () => {
+    const checkbox = checkboxRef.current.getElementsByTagName("input")[0];
+    console.log(checkbox.value);
```

Comment:
I think we can simulate the real submit event behaviour and make the example more useful by simply replacing this line with: `console.log(checkbox.checked ? checkbox.value : "");`. Maybe they find it a bit confusing that the value is still being printed even if the checkbox is unchecked.

Reply:
Done